frontend/javascripts/oxalis/view/action-bar/download_modal_view.tsx

```diff
@@ -0,0 +1,461 @@
+      const missingSelectionWarning = " Please choose a layer and a bounding box for export.";
+
+      if (selectedLayerName == null || selectedBoundingBoxID === -1) {
+        Toast.warning(basicWarning + missingSelectionWarning);
+      } else {
+        const selectedLayer = getLayerByName(dataset, selectedLayerName);
+        const selectedBoundingBox = userBoundingBoxes.find(
+          (bbox) => bbox.id === selectedBoundingBoxID,
+        );
+        if (selectedLayer != null && selectedBoundingBox != null) {
+          const layerInfos = getLayerInfos(
+            selectedLayer,
+            tracing,
+            activeMappingInfos,
+            isMergerModeEnabled,
+          );
+          await handleStartExport(
+            dataset,
+            layerInfos,
+            selectedBoundingBox.boundingBox,
+            startedExports,
+            setStartedExports,
+          );
+          Toast.success("A new export job was started successfully.");
+        } else {
+          Toast.warning(basicWarning);
+        }
+      }
+      if (!keepWindowOpen && !missingSelection) {
+        onClose();
+      }
+    }
+  };
+
+  const maybeShowWarning = () => {
+    if (activeTabKey === "download" && hasVolumeFallback) {
+      return (
+        <Row>
+          <Text
+            style={{
+              margin: "6px 12px",
+            }}
+            type="warning"
+          >
+            {messages["annotation.no_fallback_data_included"]}
+          </Text>
+        </Row>
+      );
+    } else if (activeTabKey === "python") {
+      return (
+        <Row>
+          <Text
+            style={{
+              margin: "6px 12px",
+            }}
+            type="warning"
+          >
+            {messages["annotation.python_do_not_share"]}
+          </Text>
+        </Row>
+      );
+    }
+    return null;
+  };
+
+  const handleTabChange = (key: string) => {
+    setActiveTabKey(key);
+  };
+
+  const handleCheckboxChange = (checkedValues: CheckboxValueType[]) => {
+    setIncludeVolumeData(checkedValues.includes("Volume"));
+  };
+
+  const handleKeepWindowOpenChecked = (e: any) => {
+    setKeepWindowOpen(e.target.checked);
+  };
+
+  const workerInfo = (
+    <Row>
+      <Divider
+        style={{
+          margin: "18px 0",
+        }}
+      />
+      <Text
+        style={{
+          margin: "6px 12px",
+        }}
+        type="warning"
+      >
+        {messages["annotation.export_no_worker"]}
+        <a href="mailto:hello@webknossos.com">hello@webknossos.com.</a>
+      </Text>
+    </Row>
+  );
+
+  const checkboxStyle = {
+    height: "30px",
+    lineHeight: "30px",
+  };
+
+  const authToken = useFetch(getAuthToken, "loading...", []);
+  const wkInitSnippet = `import webknossos as wk
+
+with wk.webknossos_context(token="${authToken}"):
+    annotation = wk.Annotation.download(
+        "${annotationId}",
+        annotation_type="${annotationType}",
+        webknossos_url="${window.location.origin}"
+    )
+`;
```

Comment:
Just double-checked the code, we should move the url in the context, then everything works :tada:
```suggestion
with wk.webknossos_context(
    token="${authToken}",
    url="${window.location.origin}"
):
    annotation = wk.Annotation.download(
        "${annotationId}",
        annotation_type="${annotationType}",
    )
`;
```